pkg/chaincode/approve_test.go

```diff
@@ -54,6 +55,14 @@ func NewEndorseResponse(channelName string, result string) *gateway.EndorseRespo
 	}
 }
 
+func NewEvaluateResponse(channelName string, result string) *gateway.EvaluateResponse {
```

Comment:
`channelName` is never used. The real issue is this isn't picked up by the linting so I'll fix that when I fix the linting.